frontend/src/components/episode/components/comments-list/comments-list.tsx

```diff
@@ -0,0 +1,27 @@
+import { Comment as TComment } from 'common/types/types';
+import styles from './styles.module.scss';
+import Comment from '../comment/comment';
+
+type Props = {
+  comments: TComment[];
+};
+
+const CommentsList: React.FC<Props> = ({ comments }) => {
```

Comment:
CommentsList component should return only `ul` element
move the logic to one level above